README.md

```diff
@@ -0,0 +1,18 @@
+# metrics
+Metrics gathering and monitoring for Kaazing Gateway
+
+```
+git clone https://github.com/kaazing/gateway.git
+pushd gateway
+mvn clean install -Pdocker
```

Comment:
Not sure exactly why this would be needed here. Shouldn't it be sufficient to have the metrics repo built and then running the Docker file?